### PR TITLE
Update revisionize.php

### PR DESCRIPTION
**Pull request by anantgarg**
Please update version number. I've updated mine as I din't want it to be over written.

The test case is simple:
1. Install Custom Permalinks 
2. Create a revision to a post/page which has a custom permalink
3. Make any change, and click preview, it will not work.

I've modified the revisionize code so that it does not take the custom permalink into consideration. I've also added relevant code to trash the older revision (this should ideally be a setting).

### CODE REVIEW

File: revisionize.php [hunk 1]
```diff
@@ -122,21 +123,39 @@ function create_revision($post, $is_original=false) {
...
+
+  //    flush_rewrite_rules();
+
   return $new_id;
 }
 
 function publish($post, $original) {
   if (user_can_publish_revision() || is_cron()) {
     $clone_id = create_revision($original);    // keep a backup copy of the live post.
+	wp_trash_post($clone_id); // NEW
```
Comment (jamiechong) [inline revisionize.php]: Why are you trashing the backup?

File: revisionize.php [hunk 2]
```diff
@@ -122,21 +123,39 @@ function create_revision($post, $is_original=false) {
     delete_post_meta($post->ID, '_post_original');
   }
 
+	delete_post_meta($new_id,'custom_permalink');  // NEW
+
+
+
+  wp_update_post( array(
```
Comment (jamiechong) [inline revisionize.php]: what is this for?

File: revisionize.php (hunk 1 above)
Comment (anantgarg) [reply revisionize.php]: This should actually be an optional thing. But it helps reduce the overall additional documents.

File: revisionize.php (hunk 2 above)
Comment (anantgarg) [reply revisionize.php]: The preview link wasn't working before this (in the revision). So I added this. But I'm not 100% sure about this.

**Comment by anantgarg** [comment]
> It sounds like you don't want the custom_permalink meta field to stay on the old/original post? Is that what's screwing up the permlink plugin?

If you create a revision of a page/post which has a custom permalink, the preview is not working. If you change the permalink of a revision, it is causing the live page/post to 404. So I thought that if I made it that the custom_permalink is not part of the revision, it would work. And it works fine after that.

**Comment by jamiechong** [comment]
Sorry @anantgarg I can't merge this. I'm worried it is not stable enough for the majority of people using Revisionize. Feel free to fork into your own repo to make these changes for yourself. 

If I come across this again, or if I get a chance to more throughly test the custom permalinks plugin, I'll revisit.

**Comment by anantgarg** [comment]
Sure, no problem.

**Pull request closed without merging**